scipy/interpolate/__init__.py

```diff
@@ -87,6 +87,15 @@
    splantider
    insert
 
+An alternative set of univariate spline routines:
```

Comment:
I'd put these together with the splrep list above since the objects are compatible.
Instead, the UnivariateSpline stuff should be presented as the "Alternative set".